docs/userGuide/usingPlugins.md

```diff
@@ -0,0 +1,101 @@
+<frontmatter>
+  footer: userGuideFooter.md
+  siteNav: userGuideSections.md
+</frontmatter>
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# Using Plugins
+
+A plugin is a user defined extension that can add custom features to MarkBind. Markbind plugins are `js` scripts that are loaded and run during the page generation.
+
+**WARNING:** Plugins are executable programs. This means that they might contain malicious code that may damage your computer.

```

Comment:
Add a phrase: "Plugins are executable programs **that can be written by anyone**. This means"